yaml-tests/src/test/resources/recursive-cte.yamsql

```diff
@@ -68,6 +68,49 @@ test_block:
                  {210, 20},
                  {250, 50},
                  {250, 50}]
+    -
+      - query: with recursive c1 as (
+            select id, parent from t1 where parent = -1
+            union all
+            select b.id, b.parent from c1 as a, t1 as b where a.id = b.parent) select id from c1
+      - maxRows: 1
+      - result: [{ID: 1}]
```

Comment:
there is an outstanding PR that does this stuff automatically for up/downlevel testing. So you may not need these tests in a week from now.

Reply:
Got it, I would leave it for now not to lose test coverage, and we can clean up later when we have a way to automatically test continuations in YAML.